src/lib/geoip/utils.ts

```diff
@@ -9,6 +9,8 @@ export const normalizeCityNamePublic = (name: string): string => {
 
 export const normalizeCityName = (name: string): string => normalizeCityNamePublic(name).toLowerCase();
 
+export const normalizePublicName = (name: string): string => name.toLowerCase();
```

Comment:
Probably related to the question on the dashboard side - do we need this?

Reply:
answered here: https://github.com/jsdelivr/dashboard/pull/9#discussion_r1395520408

Reply:
Ok, so on the dashboard side, we need to apply the normalization so that the values are presented to the user in the same way as from the API. I still don't understand the purpose of this function here. It's called `normalizePublicName()` but lowercasing is otherwise included only in the "internal" normalization, not public.

Reply:
Function takes "public name" as argument, and normalizes it => `normalizePublicName`. Similar to `normalizeRegionName` or `normalizeNetworkName`.

Reply:
Oh, I see, it isn't "normalize *to* public name" (what other functions there do) but "normalize *from* public name to internal one". You could probably just use `normalizeCityName()`, it'll repeat part of the process but that shouldn't cause any issues.

Reply:
This function will be called on every fetchSockets for every adopted probe so it will be great to avoid regex operations there, I can just call `toLowerCase()` in that single place where the function was used.

Reply:
Let's rename to `normalizeFromPublicName` then and move on, that should be more clear.